src/SqsPoller/SqsPoller.csproj

```diff
@@ -25,8 +25,8 @@
     
     <ItemGroup>
         <PackageReference Include="AWSSDK.SQS" Version="3.7.0.14" />
-        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="5.0.0" />
-        <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0" />
+        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
+        <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.0" />
         <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Ideally to get rid of `Newtonsoft.Json`. But it's optional

Reply:
It needs to refactor the logic of library

Reply:
next PR